### PR TITLE
Adding MobileAppTracker aka Tune library to the providers.

### PR DESCRIPTION
**Pull request by fabiojgrocha**
Added MobileAppTracker/Tune to the providers.
This implementation is not 100% complete as MobileAppTracker can receive more details, but it is enough to initialise and track events.
Also this lib has a small diference from the others, each new event is paid, that's why I've created a whitelist called "allowedEvents", so the events accepted events have to be specified while initializing the lib.

### CODE REVIEW

**Comment by orta** [comment]
Yep, this all looks :+1:

**Comment by orta** [comment]
@fabiojgrocha can you submit a note in the changelog thanking yourself please ;)

Other than that, looks good to merge.

**Comment by fabiojgrocha** [comment]
@orta first time I'm thanking to my self :D Done! thanks

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,9 @@
-language: objective-c
+brew updatelanguage: objective-c
```
Comment (orta) [inline .travis.yml]: this won't compile

**Comment by orta** [comment]
I'm not too worried about the red, I need to fix it properly sometime

**Pull request merged**